frontend/src/routes/Components/PrimaryButton.js

```diff
@@ -0,0 +1,17 @@
+import React, { Component } from "react";
+class PrimaryButton extends Component {
+  render() {
+    const primary = {
+      background: "linear-gradient(45deg, #721ebe, #3260b3)",
+      borderRadius: "0.625rem",
+      bottom:"0rem",
+      right:"15rem",
+      float:"right"
```

Comment:
what do these do? is this to center the text? or add padding? cuz i think i wouldve done "padding: 1rem" or something and "text-align: center"